spec/models/case_management_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CaseManagement, type: :model do
+  before { described_class.current = nil }
+  after { described_class.current = nil }
```

Comment:
Could add `CaseManagement.reset` and run it after each spec through `rails_helper` or something so that we don't need to worry about doing this all over the place.

Reply:
Or maybe:

```ruby
CaseManagement.current(double) do
  # Sets and clears current if given an argument and block
  # Just returns current if no args and no block
end
```

Not worth it at this point though IMO.

Reply:
Happy to leave as is for now